app/scoring/page.tsx

```diff
@@ -1,9 +1,61 @@
-export default function Scoring() {
-    var currentSelectedContestant = "Jacob"
+import { getTeamList, ITeam } from "../utils/wikiQuery"
+import { wikiUrl, getWikipediaContestantData } from "../utils/wikiFetch"
 
+export default async function Scoring() {
+
+    const wikiContestants = await getWikipediaContestantData()
+    const pageData = getTeamList(wikiContestants)
+
+    const currentSelectedContestant = "Jacob"
+
+    const numberOfRounds = pageData.props.runners.reduce(
+        (acc: number, x: ITeam) => {
+            return x.eliminationOrder > acc ? x.eliminationOrder : acc
+        }, 0)
+
+    const roundScores = []
+    for(let i = 1; i <= numberOfRounds; i++) {
+        const roundScore = pageData.props.runners.reduce(
+            (acc: number, x: ITeam) => {
+                return x.eliminationOrder === 0 || x.eliminationOrder > i ? acc + 10 : acc
+            }, 0)
+        roundScores.push(roundScore)
+    }
+
+    const weeklyScore = pageData.props.runners.reduce(
+        (acc: number, x: ITeam) => {
+            return x.isParticipating ? acc + 10 : acc
+        }, 0)
+
+    const reverseTeamsList = [...pageData.props.runners].reverse()
+    let grandTotal = 0
+    let currentWeek = 0
     return (
         <div>
             <h1 className="text-2xl text-center">Current Scoring for {currentSelectedContestant}</h1>
+            <br/>
+            <div className="text-center">
+                {roundScores.map(s => {
+                    grandTotal += s
+                    currentWeek++
+
+                    return (<>
```

Comment:
Oh my god. For some reason I thought this was a react project. 🙃
Sorry, was just looking at some react stuff.

Reply:
Oh no, you are all good here. Next is built on top of React AFAIU, so that is perfectly reasonable mistake to make, plus I think this suggestion is not only a good one but actually a necessary one since both this fragment and the one below should have a `key` prop and I think the only way to do that and not confuse the heck out of the compiler is to clearly mark them as `Fragment`s